source/design-system.js

```diff
@@ -1,17 +1,17 @@
 /**
+ * Design System
  * The kitchen sink of all design components
  */
 
-import breadcrumb from 'atoms/breadcrumb';
-import button from 'atoms/button';
-import image from 'atoms/image';
-import icon from 'atoms/icon';
-import listItem from 'atoms/list-item';
-import card from 'molecules/card';
-import jumbotron from 'molecules/jumbotron';
+import * as breadcrumb from 'atoms/breadcrumb';
+import * as button from 'atoms/button';
```

Comment:
Considering how simple our templates for `js` behavior in components is, does it make sense to do a wildcard import? Or are you trying to future-proof these?

Reply:
this wildcard import ends up with the same result as the previous syntax. with how the js files are now broken up into discrete exports instead of one total object, we want to be sure we have the entire thing when we pull it into the full design system

Reply:
This is how we get an entire component into a var, including all it's exported functions. So we can now throw around button and have a safe assumption it will always have `.enable()`, `.disable()` etc